ProgramFlow/ProgramFlow/Flow.cs

```diff
@@ -18,30 +12,58 @@ public static class Flow
         /// <returns><b>true</b> if <paramref name="value"/> is <b>false</b>; <b>false</b> if <paramref name="value"/> is <b>true</b>.</returns>
         public static bool InvertBoolean(bool value)
         {
-            // TODO: Fix this method so it passes all associated tests.
```

Comment:
Thank you for removing the TODO comments since they are unnecessary after you fixed the issues.